app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt

```diff
@@ -60,6 +62,7 @@ class AddReviewActivity : AppCompatActivity() {
 
         ratingBar = findViewById(R.id.reviewRatingBar)
```

Comment:
The two next lines have ids with snake_case. We will need to decide whether snake_case or camelCase is appropriate for ids, but apart from that everything seems ok.

Reply:
Well someone can clean all naming issues and do a PR.

Reply:
I will try to do it within this PR.

Reply:
Done, you can check the commit.

Reply:
Perfect!